scss/_layouts_docs.scss

```diff
@@ -0,0 +1,31 @@
+@mixin vf-l-docs {
+  .l-fixed-width--with-sidebar {
+    @extend %vf-row;
+  }
+
+  .l-sidebar,
+  .l-content {
+    @extend %span-full-grid-on-mobile;
+    @extend %span-full-grid-on-tablet;
+    @extend %display-block;
+  }
+
+  @media (min-width: $threshold-6-12-col) {
+    .l-fixed-width--with-sidebar {
+      grid-template-areas: 'side side side main main main main main main main main main';
```

Comment:
Can you use `repeat` here? For example:
```css 
.l-fixed-width--with-sidebar {
  grid-template-areas: "repeat(3, 'side') . repeat(8. 'main')";
}
```

Reply:
Not really, I tried that. It's only for `grid-template-columns` or `grid-template-rows` where these functions are available.
With `grid-template-areas` we use the row/col pattern defined (12 columns in our case) and we need to fill it with areas. To span things across cells you need to repeat in the template.